hawkular-dmr-client/src/main/java/org/hawkular/dmr/api/DmrConstants.java

```diff
@@ -0,0 +1,483 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.dmr.api;
+
+/**
+ * @author <a href="https://github.com/ppalaga">Peter Palaga</a>
+ */
+public interface DmrConstants  {
```

Comment:
OK, let's consume their's `ModelDescriptionConstants`. It is not an interface as mine `DmrConstants` to be able to "import via implements" but nevermind. 

Thinking again about possible future changes in `ModelDescriptionConstants`, they'll most probably just be adding new constants and never remove one, which is something we can live with in the long term. I mean there probably comes a day when we'll have to support multiple EAP/WF versions and we'll be able to import the `ModelDescriptionConstants` just from the newest AS. WF core team should not cause any problems to us if they behave responsibly.